Makefile

```diff
@@ -1,10 +1,13 @@
+FILE=schemas_version.txt
```

Comment:
As we already have a .python-version file, it would make sense to follow the same form for our schemas and call this .schemas-version

Reply:
Done, thanks!